pyproject.toml

```diff
@@ -26,7 +26,6 @@ classifiers = [
 ]
 
 [tool.poetry.dependencies]
-async-timeout = { version = ">=3.0.0", python = "<3.11" }
```

Comment:
This is a runtime dependency, so we can't remove this.

It is only building the docs that requires Python >=3.11. Bleak itself still supports down to 3.9.